util.js

```diff
@@ -943,6 +943,14 @@ export const handleDropJsonItem = async (item) => {
   }
   return null;
 };
+export const handleBlobUpload = async (name, blob, progress) => {
+  const formData = new FormData();
+  formData.append(name, blob, name);
+  const hashes = await doUpload(`https://ipfs.webaverse.com/`, formData, progress);
+
+  const rootDirectoryHash = hashes.length > 0 ? hashes[0].hash : null;
```

Comment:
This isn't used?

Reply:
As you can see, this value is used in the next line.

Reply:
As you can see, this value is used in the next line.